packages/ListBoxBrowser/README.md

```diff
@@ -0,0 +1,173 @@
+# ListBoxBrowser
+
+It is a React component that allows users to select one or multiple options using two columns, similar to the experience of browsing a file manager in any giving Operative System.
+
+## Installation
+
+```sh
+$ npm install @paprika/listbox-browser
+```
+
+or
+
+```sh
+$ yarn add @paprika/listbox-browser
+```
+
+## Props
+
+```
+browserTitle
+children
+data
+defaultSelectedOptions
+defaultSelectedView
+hasBreadcrumb
+hasError
+height
+isMulti
+isParentSelectable
+onChange
+rootTitle
+```
+
+## Usage
+
+The `<ListBoxBrowser />` is primarily an uncontrolled component which received a `data` prop allowing the consumer to set the initial state.
+
+Also, has two functions to set defaults for selected options and a default selected view for the browser column.
+
+The `onChange` prop receives a parameter with an array of the selected options.
+
+### Data prop
+
+The most important prop for the `<ListBoxBrowser />` is the `data` prop, which initializes the state for the component.
+
+The `data` prop has a shape of an array of objects where each object requires to have at least a `label` attribute on it, and at least one of those items should include an `options` property.
+
+### Data prop shape
+
+```js
+{
+	label: "require attribute",
+	options: [{}{}{}], // more object with the same shape
+}
+```
+
+The **object** can have an options attribute, which is an array of the same kind of objects.
+
+A data prop with **options** looks like:
+
+```js
+const data = [
+   {
+        label: "NBA",
+        options: [{ label: "Toronto Raptors"}, {...}]
+   },
+   {
+        label: "NHL",
+        options: [{ label: "Vancouver Canucks"}, {...}]
+   },
+   {
+        label: "MLS",
+        options: [{ label: "Vancouver Whitecaps"}, {...}]
+   },
+]
+```
+
+Recursively the component runs over all the options on the array and assembles the appropriated UI for the user.
+
+**NOTE** You can add extra properties to the objects; those are and pass to you back on the `onChange` function or when using `defaultSelectedOptions`, `defaultSelectedView` functions.
```

Comment:
This one i'm not really sure about the grammar on this part
"those are and pass to you back on the `onChange` function or when using `defaultSelectedOptions`, `defaultSelectedView` functions.".

Maybe pro @alexzherdev can chime in with his english skills :)